app/src/main/res/layout/news_item.xml

```diff
@@ -0,0 +1,60 @@
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:padding="16dp">
+
+
+    <ImageView
+        android:id="@+id/imageView"
```

Comment:
Please, use the same rule for naming. If you decided to use C-style notation, then continue using it across your layouts. Dont's switch from it to camel case and back. So, `imageView` will be `image_view`. If you want to keep camel case, then keep it and rename others, so `home_view` will become `homeView`. Please, make a decision on this. We usually recomend to use C-style for identifiers.